### PR TITLE
few tiny changes required for golang 1.18

### PR DESCRIPTION
**Pull request by iamniting**
Signed-off-by: Nitin Goyal <nigoyal@redhat.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -206,7 +206,7 @@ ifneq ($(CONTROLLER_GEN_VERSION), $(shell controller-gen --version | awk -F ":"
 	CONTROLLER_GEN_TMP_DIR=$$(mktemp -d) ;\
 	cd $$CONTROLLER_GEN_TMP_DIR ;\
 	go mod init tmp ;\
-	go get sigs.k8s.io/controller-tools/cmd/controller-gen@$(CONTROLLER_GEN_VERSION) ;\
+	go install sigs.k8s.io/controller-tools/cmd/controller-gen@$(CONTROLLER_GEN_VERSION) ;\
```
Comment (umangachapagain) [inline Makefile:209]: You can remove lines 206-208 and 210. `go install` doesn't need those.

Comment (umangachapagain) [reply Makefile:209]: Also, update kustomize install.

**Review by umangachapagain — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (iamniting) [reply Makefile:206]: done

**Review by jarrpa — APPROVED** [review]
/lgtm

**Review by jarrpa — APPROVED** [review]
/lgtm

**Pull request merged**